docs/guide/deployment/certificate-authority.md

```diff
@@ -0,0 +1,120 @@
+# Certificate authority
+
```

Comment:
Pls describe purpose of service.
This service is used to sign certificates for devices and plgd-dev/sdk (https://github.com/plgd-dev/sdk) client. 

BTW: During onboarding device via SDK (at first SDK gets OCF identity certificate for self), SDK ask the service to sign CSR (certificate signed request) for the device and when it is success the service returns certificate chain.